jtransc-rt/src/java/lang/StringBuilder.java

```diff
@@ -35,7 +35,7 @@
 	"public function setStr(str:String) { this.str2 = str; buffer2 = new MyStringBuf(); buffer2.add(str); return this; }",
 })
 @JTranscAddMembers(target = "as3", value = "public var _str: String = '';")
-@JTranscAddMembers(target = "dart", value = "StringBuffer _buffer = new StringBuffer();")
+@JTranscAddMembers(target = "dart", value = "StringBuffer __buffer = new StringBuffer();")
```

Comment:
This conflicts with the field char[] buffer ( renamed to _buffer after generation. )